python_modules/dagster/dagster/_core/definitions/definitions_loader.py

```diff
@@ -64,6 +64,11 @@ def get(cls) -> "DefinitionsLoadContext":
             )
         return DefinitionsLoadContext._instance
 
+    @classmethod
```

Comment:
@smackesey  this API should be structured so this is not necessary. Recommendation is `get` defaults to an initializing context with no load data.

Reply:
Yep finding a need for that myself PR up in a minute.

Reply:
https://github.com/dagster-io/dagster/pull/24626